src/DependencyInjection/Configuration.php

```diff
@@ -32,13 +32,30 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->scalarNode('table_prefix')->defaultValue('_acb_')->end()
             ->end();
 
+        $this->addPublishableNode($rootNode);
         $this->addSecurityNode($rootNode);
         $this->addEnabledComponentsNode($rootNode);
         $this->addUserNode($rootNode);
 
         return $treeBuilder;
     }
 
+    private function addPublishableNode(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('publishable')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('permission')
+                            ->cannotBeEmpty()
+                            ->defaultValue(sprintf('is_granted(ROLE_ADMIN)'))
```

Comment:
if this default value is set, should we be adding it into the recipe. I think it's better to make sure a user defines this role explicitly and so if there is a default value in the config, I have a feeling I had an old recipe that was rejected because it didn't need to be specified. Perhaps remove the default value and just make it required and keep it in the recipe?